x-pack/packages/security/plugin_types_public/src/plugin.ts

```diff
@@ -14,6 +14,8 @@ import type { UserProfileAPIClient } from './user_profile';
 export interface SecurityPluginSetup {
   /**
    * Exposes authentication information about the currently logged in user.
+   *
+   * @deprecated in favor of Core's `security` service
```

Comment:
```suggestion
   * @deprecated. Use Core's `security` service instead.
```

Reply:
Same suggestion for the other deprecation notices.

Reply:
IIRC You can't have a `.` just after the `@deprecated`, it breaks the annotation

Reply:
In that case, ignore the suggestion.

Reply:
I think it still may be relevant (without the dot).

How do IDEs show the deprecation?

Is it `[Deprecated] in favor of...` (deprecated as a badge), or a simpler `Deprecated in favor of...`?